pkg/client/actionclient_test.go

```diff
@@ -470,7 +470,7 @@ var _ = Describe("ActionClient", func() {
 			}
 			patch, patchType, err := createPatch(o1, o2)
 			Expect(err).To(BeNil())
-			Expect(string(patch)).To(Equal(`{}`))
+			Expect(string(patch)).To(Equal(``))
```

Comment:
Now that patch is `nil`, perhaps just directly check that its `nil` instead of converting to a string?
```suggestion
			Expect(patch).To(BeNil())
```